backends/kafka/read.go

```diff
@@ -63,11 +70,34 @@ func (k *Kafka) Read() error {
 		}
 
 		data, err := reader.Decode(k.Options, k.MsgDesc, msg.Value)
+
 		if err != nil {
 			return err
 		}
 
-		printer.PrintKafkaResult(k.Options, count, msg, data)
+		if k.Options.ReadLag && msg.Partition != lastPartitionProcessed {
+
+			lastPartitionProcessed = msg.Partition
+
+			lagConn, err = NewKafkaLagConnection(k.Options)
+
+			if err != nil {
```

Comment:
I would print a warning here - just to make it apparent that a connection failed (maybe even debug?).

Reply:
Ended up with debug :)